### PR TITLE
docs: Add an RFC process for new changes

### PR DESCRIPTION
**Pull request by losvedir**
Adds the process that the TRC team discussed for its `gtfs_creator` app, inspired by rust-lang's RFCs. But the process can be generally applicable to any team's apps, so let's try doing it here!

### CODE REVIEW

File: rfcs/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# RFCSs
+
+The RFC process here is inspired by [that of the Rust programming
+language](https://github.com/rust-lang/rfcs).
+
+The process is intended to provide TRC team members a venue in which to propose substantial
```
Comment (paulswartz) [inline rfcs/README.md]:
```suggestion
The process is intended to provide CTD team members a venue in which to propose substantial
```

File: rfcs/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  accordingly if the RFC is accepted.
+* Fill in the RFC. Put care into the details: RFCs that do not present convincing motivation,
+  demonstrate lack of understanding of the design's impact, or are disingenuous about the drawbacks
+  or alternatives tend to be poorly-received.
+* Submit a pull request. As a pull request the RFC will receive design feedback from the team, and
+  the author should be prepared to revise it in response.
+* Now that your RFC has an open pull request, use the issue number of the PR to update your 0000-
+  prefix to that number.
+* Assign the RFC to the tech lead, and announce it in the Slack channel for all the team to review.
+* RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown.
```
Comment (paulswartz) [inline rfcs/README.md]: Maybe say here (or somewhere else) that it's worth drafting and getting feedback on an RFC even before formally proposing it?

File: rfcs/README.md (hunk 1 above)
Comment (losvedir) [reply rfcs/README.md]: Just noticed the "Commit suggestion" feature here that I could have used. Nifty!

File: rfcs/README.md (hunk 2 above)
Comment (losvedir) [reply rfcs/README.md]: Added a final paragraph to the "Before creating an RFC" section with this.

**Comment by losvedir** [comment]
Addressed those comments, and I guess changed `*` to `-` upon saving in my editor.

**Review by paulswartz — APPROVED** [review]
🍰

**Pull request merged**